src/Library/NewsWidget.php

```diff
@@ -156,7 +156,12 @@ public function getPosts( $limit ) {
 		$body  = wp_remote_retrieve_body( $request );
 		$posts = json_decode( $body );
```

Comment:
`json_decode()` is returning an object or false here.  It is later expected to be an array.

Reply:
The actual return in this case is an array of objects.